paper/src/main/java/co/aikar/commands/PaperAsyncTabCompleteHandler.java

```diff
@@ -70,7 +70,8 @@ public void onAsyncTabComplete(AsyncTabCompleteEvent event) {
             }
             event.getCompletions().addAll(results);
             event.setHandled(true);
-        } catch (Exception ignored) {}
+        } catch (Exception ignored) {
```

Comment:
Can't we keep brackets on one line for cases like this?

Reply:
That should be fixed with https://github.com/aikar/commands/pull/116/commits/919f940a4f39255490e9691717fbd360b8923083 and https://github.com/aikar/commands/pull/116/commits/14a4487337471e91791328877003d909d3be31b3